testing/php7-phalcon/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+pkgname=php7-phalcon
+_pkgext=phalcon
+_pkgreal=cphalcon
+pkgver=3.3.2
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="High performance, full-stack PHP framework delivered as a C extension"
+url="https://github.com/phalcon/cphalcon"
+arch="all"
+options="!check" # there is no module tests
```

Comment:
not exactly true https://travis-ci.org/phalcon/cphalcon

Reply:
better to said `require extra services to run tests` https://github.com/phalcon/cphalcon/blob/master/.travis.yml#L37